pandas/tests/io/pytables/test_compat.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+import pandas as pd
+from pandas.util.testing import assert_frame_equal
+
+from pandas.tests.io.test_pytables import ensure_clean_path
+
+
+tables = pytest.importorskip('tables')
+
+
+@pytest.fixture
+def pytables_hdf5_file(tmp_path):
```

Comment:
I don't think you need this fixture anymore

Reply:
> I don't think you need this fixture anymore 

Thanks man. I hope one of the linters would have told me! :-)

Reply:
yeah i don’t think we have that check in linting :)